PowerPointLabs/PowerPointLabs/ELearningLab/Service/ELearningService.cs

```diff
@@ -70,8 +70,19 @@ private void CreateAppearEffectAnimation(PowerPointSlide slide, ExplanationItem
             List<Effect> effects = new List<Effect>();
             if (selfExplanationItem.IsVoice)
             {
-                Effect effect = AudioService.CreateAppearEffectAudioAnimation(slide, selfExplanationItem.CaptionText, selfExplanationItem.VoiceLabel,
-                    selfExplanationItem.ClickNo, selfExplanationItem.tagNo, isSeparateClick);
+                Effect effect;
+                if (selfExplanationItem.IsVoiceLabelInvalid)
+                {
+                    string defaultVoiceLabel = string.Format(ELearningLabText.AudioDefaultLabelFormat, AudioSettingService.selectedVoice.VoiceName);
+                    effect = AudioService.CreateAppearEffectAudioAnimation(slide, selfExplanationItem.CaptionText, defaultVoiceLabel,
+                     selfExplanationItem.ClickNo, selfExplanationItem.tagNo, isSeparateClick);
```

Comment:
the indentation looks a bit weird here (1 space only)

Reply:
Thanks for noticing! Indented properly.